### PR TITLE
Ra 1261

### PR DESCRIPTION
**Pull request by ylatuya**
#### Things to review:
- [ ] Unit tests
- [ ] No hardcoded resources (strings, sizes, colors...)
- [ ] Sorted usings
- [ ] Add summaries where needed 

If needed, you can add new checkboxes to https://github.com/fluendo/VAS/blob/master/.github/PULL_REQUEST_TEMPLATE.md

### CODE REVIEW

File: VAS.Drawing/CanvasObjects/Statistics/PercentCircularChartView.cs [hunk 1]
```diff
@@ -39,11 +42,19 @@ public class PercentCircularChartView : CanvasObject, ICanvasObjectView<PercentC
 				viewModel = value;
 				if (viewModel != null) {
 					viewModel.PropertyChanged += HandlePropertyChanged;
-					CallRedraw ();
+					viewModel.Sync ();
 				}
 			}
 		}
 
+		double Percent {
```
Comment (dfernandezramos) [inline VAS.Drawing/CanvasObjects/Statistics/PercentCircularChartView.cs]: Document properties

File: VAS.Drawing/CanvasObjects/Statistics/PercentCircularChartView.cs [hunk 2]
```diff
@@ -23,10 +23,13 @@
 
 namespace VAS.Drawing.CanvasObjects.Statistics
 {
-	[System.ComponentModel.ToolboxItem (true)]
```
Comment (dfernandezramos) [inline VAS.Drawing/CanvasObjects/Statistics/PercentCircularChartView.cs]: Document class

File: VAS.Drawing/CanvasObjects/Statistics/PercentCircularChartView.cs [hunk 3]
```diff
@@ -62,52 +73,37 @@ public override void Draw (IDrawingToolkit tk, Area area)
...
-		}
-
-		void CallRedraw ()
-		{
-			if (App.IsMainThread) {
-				ReDraw ();
-			} else {
-				App.Current.GUIToolkit.Invoke ((sender, e) => ReDraw ());
-			}
+			Percent = viewModel.PercentValue;
```
Comment (dfernandezramos) [inline VAS.Drawing/CanvasObjects/Statistics/PercentCircularChartView.cs]: Don't we need to check if PercetValue needs sync? In this case each time a property changes is being updated.

File: VAS.Drawing.Cairo/WidgetWrapper.cs [hunk 4]
```diff
@@ -104,6 +104,16 @@ protected override void DisposeManagedResources ()
 			}
 		}
 
+		Rectangle WidgetArea {
```
Comment (dfernandezramos) [inline VAS.Drawing.Cairo/WidgetWrapper.cs]: Document property

File: VAS.Drawing/CanvasObjects/Statistics/PercentCircularChartView.cs (hunk 1 above)
Comment (ylatuya) [reply VAS.Drawing/CanvasObjects/Statistics/PercentCircularChartView.cs]: It's private

File: VAS.Drawing/CanvasObjects/Statistics/PercentCircularChartView.cs (hunk 3 above)
Comment (ylatuya) [reply VAS.Drawing/CanvasObjects/Statistics/PercentCircularChartView.cs]: Right

File: VAS.Drawing/CanvasObjects/Statistics/PercentCircularChartView.cs (hunk 2 above)
Comment (ylatuya) [reply VAS.Drawing/CanvasObjects/Statistics/PercentCircularChartView.cs]: Not mine but I will

File: VAS.Drawing.Cairo/WidgetWrapper.cs (hunk 4 above)
Comment (ylatuya) [reply VAS.Drawing.Cairo/WidgetWrapper.cs]: private one

**Pull request merged**